packages/core/src/types.ts

```diff
@@ -121,10 +121,12 @@ export type ComponentRegistration = {
   component: React.ElementType;
   definition: ComponentDefinition;
   options?: {
+    /**
+     * If true, the component receives the optional boolean property `isInExpEditorMode` to
+     * render different content between editor and delivery mode.
+     */
+    enableCustomEditorView?: boolean;
     wrapComponent?: boolean;
-    /** @deprecated use wrapContainer instead */
-    wrapContainerTag?: keyof JSX.IntrinsicElements;
-    wrapContainer?: keyof JSX.IntrinsicElements | React.ReactElement;
```

Comment:
Why are we removing the `wrapContainer`? This is not deprecated from what I understand. It is also mentioned in the docs: https://www.contentful.com/developers/docs/experiences/component-wrapper/#disable-the-wrapping-container

Reply:
I guess we should then update our docs as this is not supported by our code. If you search for `wrapContainer` you see that it's only defined in types but never consumed

Reply:
Hmm, maybe we should double check with altitude in that case, just to understand the reasons for not being implemented.

Reply:
Alright, will remove it again from this PR and ask them 👍🏻